src/cyclonedds/cyclonedds/tools/pubsub/parse_qos.py

```diff
@@ -0,0 +1,161 @@
+from dataclasses import fields
+from typing import Sequence, List, Union
+
+from cyclonedds.qos import BasePolicy, Qos, Policy
+from cyclonedds.util import duration
+
+
+dds_infinity = 9223372036854775807
```

Comment:
I'm not sure I like the way this is redefined. Can we get it from the `_clayer`?

Reply:
`dds_infinity` is sadly not included as exported symbol in CycloneDDS. I think the most elegant way is to add `from cyclonedds.util import duration; duration(infinite=True)`